docker.go

```diff
@@ -178,22 +178,22 @@ func (p Plugin) Exec() error {
 
 	// add base image docker credentials to the existing config file, else create new
 	if p.BaseImagePassword != "" {
-		json, err := setDockerAuth(p.Login.Username, p.Login.Password, p.Login.Registry,
-			p.BaseImageUsername, p.BaseImagePassword, p.BaseImageRegistry)
-		if err != nil {
-			return fmt.Errorf("Failed to set authentication in docker config %s", err)
-		}
-		os.MkdirAll(dockerHome, 0600)
-		path := filepath.Join(dockerHome, "config.json")
-		file, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
-		if err != nil {
-			return fmt.Errorf("Error opening config.json: %s", err)
-		}
-		defer file.Close()
-		_, err = file.Write(json)
+		cmd := exec.Command(
```

Comment:
refactor this to use `commandLogin`. Create a new login from baseImage credentials

Reply:
Done.